pangeo_forge/storage.py

```diff
@@ -0,0 +1,65 @@
+import os
+from contextlib import contextmanager
+from dataclasses import dataclass
+
+import fsspec
+
+
+@dataclass
+class Target:
```

Comment:
Maybe this should be called a `ZarrTarget`?

Reply:
Maybe `MapperTarget`? There is nothing zarr specific about it...